include/pmacc/mappings/kernel/AreaMappingMethods.hpp

```diff
@@ -30,12 +30,44 @@ namespace pmacc
      * Helper class for AreaMapping.
      * Provides methods called by AreaMapping using template specialization.
      *
-     * @tparam areaType the area to map to
+     * @tparam AreaType the area to map to
      * @tparam DIM dimension of the mapping
+     * @tparam Origin Return supercellIdx wrt specified origin
      */
-    template<uint32_t areaType, unsigned DIM>
+    template<unsigned AreaType, unsigned DIM, uint32_t Origin = GUARD>
```

Comment:
`AreaType` the naming was before confusing and we should change it. It is not a type.

The naming rules are `uint32_t T_area` and `T_origin` if it is not a type.
Template types `typename T_CameCase` are always start with capital letter.

see https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#16-template-declarations-and-specializations

Reply:
Renamed to T_area